directdemod/json_parser.py

```diff
@@ -0,0 +1,126 @@
+'''
+Object for json manipulation
```

Comment:
If possible put it in misc.py, I just want the directDemod folder to be mostly "comm" stuff.

Reply:
Okey.